### PR TITLE
this.toggle logic

### PR DESCRIPTION
**Pull request by d4rkstar**
Hello I fixed the pg-toggleable javascript. The toggle switch stuck after first change 😡

This PR seems to solve

### CODE REVIEW

**Comment by d4rkstar** [comment]
And.. i added flatpickr inside package.json because it seems to be missing.

**Comment by luanfreitasdev** [comment]
Thanks for submitting the PR.

I noticed you didn't run the `npm run build` command, right?

File: package.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
     },
     "homepage": "https://github.com/Power-Components/livewire-powergrid#readme",
     "dependencies": {
+        "flatpickr": "^4.6.13",
```
Comment (luanfreitasdev) [inline package.json:24]: flatpickr should not be loaded inside the powergrid. It has to be loaded from the outside

**Review by luanfreitasdev — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (d4rkstar) [reply package.json:24]: Hello,
when i ran the "npm run build" command for the first time, i got the error visible in the screenshot

image

I tried to imagine that some dependency was missing. In fact, the build does not happen without adding the flatpickr library. 

Do you think it is appropriate to leave flatpickr but put it in the devDependencies or remove it?

Comment (luanfreitasdev) [reply package.json:24]: Put it as devDependencies, no problem

Comment (d4rkstar) [reply package.json]: Done! :sweat_smile:   
Let me know if now is ok

**Review by luanfreitasdev — APPROVED** [review]

**Pull request merged**